src/editor/components/modals/ScreenshotModal/ScreenshotModal.component.jsx

```diff
@@ -41,23 +42,16 @@ const getSceneName = (scene) => {
 };
 
 function ScreenshotModal() {
+  // Get the entity that has the screentock component
+  const getScreentockEntity = () => {
+    const screenshotEl = document.getElementById('screenshot');
+    screenshotEl.play();
```

Comment:
Maybe call play only if paused, otherwise you may register listeners again and have issues.
```js
if (!screenshotEl.isPlaying) {
  screenshotEl.play();
}
```

Reply:
509990df0bb8ba5ca6fea2c7794d411669ce7094